File: test/fixtures/python/get_with_form.py
import requests

files = {
    'from': (None, 'test@tester.com'),
    'to': (None, 'devs@tester.net'),
    'subject': (None, 'Hello'),
    'text': (None, 'Testing the converter!'),
}

response = requests.post('http://localhost:28139/v3', files=files, auth=('test', ''))

File: test/fixtures/csharp/get_with_form.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, "http://localhost:28139/v3");

request.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(System.Text.ASCIIEncoding.ASCII.GetBytes("test:")));


MultipartFormDataContent content = new MultipartFormDataContent();
content.Add(new StringContent("test@tester.com"), "from");
content.Add(new StringContent("devs@tester.net"), "to");
content.Add(new StringContent("Hello"), "subject");
content.Add(new StringContent("Testing the converter!"), "text");
request.Content = content;

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();
